packages/docs/src/examples/va-image/ErrorSlot.vue

```diff
@@ -1,12 +1,9 @@
 <template>
   <div class="row">
     <div class="flex md6 lg4">
-      <va-image
-        style="min-height: 300px;"
-        src="wrong-image-path"
-      >
+      <va-image style="min-height: 300px;" src="wrong-image-path">
         <template #error>
-          Image not found! :(
```

Comment:
You can replace with "Not found" and make generic.
Please remember, that every line you add has to be translated and supported.

Reply:
moved to generic translations